test/functional/test_framework/test_framework.py

```diff
@@ -102,7 +102,10 @@ def __init__(self):
         self.bind_to_localhost_only = True
         self.set_test_params()
         self.parse_args()
-        self.rpc_timeout = int(self.rpc_timeout * self.options.factor) # optionally, increase timeout by a factor
+        if self.options.timeout_factor == 0 :
+            self.rpc_timeout = 99999
```

Comment:
```suggestion
            self.options.timeout_factor = 99999
```

This should apply to all timeouts, otherwise the tests will crash in `sync_*` calls

Reply:
Right, I missed that part. Fixed.